app/controllers/userrole.controller.js

```diff
@@ -2,25 +2,38 @@ const db = require("../models");
 const UserRole = db.userRole;
 const Role = db.role;
 
-exports.findAllForUser = async (req, res) => {
+const getAllForUser = async (req, res) => {
   const { userId } = req.params;
-  await UserRole.findAll({
-    where: { userId: userId },
-    include: [
-      {
-        model: Role,
-        required: true,
-        as: "role",
-      },
-    ],
-  })
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      res.status(500).send({
-        message:
-          err.message || "Some error occurred while retrieving user roles.",
-      });
-    });
+
+  try {
+    const userRoles = await db.sequelize.query(
```

Comment:
We need to talk about this

Reply:
This sounds like a disappointed dad. I'm scared.

Reply:
Sorry to freak you out lol